static/scss/partials/modals.scss

```diff
@@ -0,0 +1,151 @@
+.modal-message {
+    @include text-body-sm;
+    color: $color-grey-60;
+    margin: 0 auto $spacing-sm auto;
+
+    strong {
+        word-break: break-word;
+    }
+    
+    &.warning-text {
+        color: $color-error;
+    }
+}
+
+.alias-to-delete {
+    display: inline;
+    word-break: break-all;
+
+    &.demi {
+        display: inline-block;
+        margin: 0 auto $spacing-md;
+        // margin: 0 auto $spacing-md 0;
+        color: $color-grey-60;
+        max-width: $content-xs;
+        text-overflow: ellipsis;
+        overflow: hidden;
+        word-wrap: unset;
+    }
+}
+
+// Modal Protocol Refactor
+.c-modal-wrapper {
+    position: fixed;
+    display: none;
+    background: rgba(0, 0, 0, .4);
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    z-index: 10;
+    justify-content: center;
+    flex-direction: column;
+
+    &.is-visible {
+        display: flex;
+    }
+}
+
+.c-modal {
+    background: $color-white;
+    max-width: $content-sm-md;
+    border-radius: $border-radius-md;
+    position: absolute;
+    left: $spacing-xl;
+    right: $spacing-xl;
+    margin: auto;
+    box-shadow: $box-shadow-sm;
+    overflow: hidden;
+    padding: $spacing-md;
+}
+
+.c-modal-header {
+    border-radius: $border-radius-md;
+    background-color: $color-grey-05;
+    padding: $spacing-lg $spacing-md;
+}
+
+.c-modal-header-content {
+    @include text-title-3xs;
+    
+    h3 {
+        @include text-title-xs;
+        color: $color-grey-40;
+        margin: $spacing-sm 0;
+        font-weight: 700;
```

Comment:
Do we use font-weight numbers (`700`) or names (`bold`)?

Reply:
I've personally been using font-weight numbers since there's usually a CSS property for it in Figma.

Reply:
+1 for numbers. Was curious how you were handling it.